config/core/test-configs.yaml

```diff
@@ -61,8 +61,8 @@ file_systems:
 
   debian_buster-ltp_nfs:
     type: debian
-    ramdisk: 'buster-ltp/20210520.0/{arch}/initrd.cpio.gz'
-    nfs: 'buster-ltp/20210520.0/{arch}/full.rootfs.tar.xz'
+    ramdisk: 'buster-ltp/20210601.0/{arch}/initrd.cpio.gz'
```

Comment:
This won't work because it's based on the production URL.  You also need to define the base URL for staging at the top of the file.

Reply:
I see. I changed it following you patch suggestion.